python/README.md

```diff
@@ -0,0 +1,34 @@
+# Python CSFLE Example
+
+## Steps
+
+1. Start a locally running `mongod` version >= 4.2 running on port 27017
+2. It is recommended you create a local environment to encapsulate the
+   dependencies. We use **pyenv**
+
+   ```python
+   pyenv virtualenv 3.7.3 csfle
+   pyenv local csfle
+   pip install -r requirements.txt
+   ```
+
```

Comment:
Would it be helpful to have them locate and specify the path for mongocryptd on their system?

Reply:
addressed with usage string